FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m

```diff
@@ -484,7 +484,7 @@ - (void)verifyClientAndSendVerificationCodeToPhoneNumber:(NSString *)phoneNumber
                           FIRStartMFAEnrollmentRequest *request =
                               [[FIRStartMFAEnrollmentRequest alloc]
                                        initWithIDToken:IDToken
-                                        enrollmentInfo:startMFARequestInfo
+                                   phoneEnrollmentInfo:startMFARequestInfo
```

Comment:
It is clearer in terms of code readability since writing `enrollmentInfo` could be confusing to somebody using the library as more MFA features get added.

Reply:
Moving this change out to a different PR seems more feasible. Reverting this for now.